### ISSUE TITLE
MR2 SW20 Braking Understeer

### ISSUE THREAD

**Issue by Jinxmaster1** [TheTofuShop/Vehicles#11]
🚗 **What is the spawn model or name of the vehicle?** (e.g. zn6 or Toyota 86)

MR2SW20

🔧 **What exactly needs to be changed?** (use https://gtamods.com/wiki/Handling.meta for reference)

Uncertain as to the cause but maybe it is fBrakeBiasFront fSteeringLock or a combination of
 parameters.

Basically the car has too much foot brake induced understeer. In this car if you try to brake as you turn you will lose ~75%-90% of your steering angle. This is excessive and makes the car irritating to drive.

**Comment by TheAussie** [comment]
@Jinxmaster1 this is generally what happens with every car irl, this is why you don't turn under braking, you do your braking on the straight before the corner.

**Comment by TheAussie** [comment]
I have tested, and the understeer doesn't only occur during braking.

**Comment by TheAussie** [comment]
@Jinxmaster1 i think... i somewhat fixed it.

**Comment by Jinxmaster1** [comment]
Definite improvement. I do feel like the SW20 should be more of a battle against oversteer (to match it's reputation for snap oversteer) but the current state is definitely good enough for now to the point where it drives fine for now just the character is off for the irl car it represents.

**Comment by TheAussie** [comment]
> Definite improvement. I do feel like the SW20 should be more of a battle against oversteer (to match it's reputation for snap oversteer) but the current state is definitely good enough for now to the point where it drives fine for now just the character is off for the irl car it represents.

Yeah I might have another look at it later, thanks for the feedback.

**Comment by SiirToast** [comment]
Totally aggree with @Jinxmaster1 , car should be a continuous struggle to keep the vehicle on the line without losing control. Overall, the improvement is very good compared to the old handling and we're set on the right track
